examples-next/ecommerce/mutations.ts

```diff
@@ -66,14 +66,16 @@ export const extendGraphqlSchema = graphQLSchemaExtension({
 
         // 5. create the Order
         console.log('Creating the order');
-        const order = await context.lists.Order.createOne({
-          data: {
-            total: charge.amount,
-            charge: `${charge.id}`,
-            items: { create: orderItems },
-            user: { connect: { id: userId } },
-          },
-        });
+        const order = await context
```

Comment:
For the example, I'd prefer to see the context created separately from the order. I can imagine people copying+pasting this pattern and creating a new context for each items API operation, so let's clarify that into two steps.

Reply:
Experimented with this in #4463